projects/app/test/client/query.test.ts

```
@@ -79,21 +79,55 @@ await test(`${computeSkillReviewQueue.name} suite`, async () => {
       `he:丿:slash`,
     ]);
   });
+
+  await test(`doesn't get stuck reviewing the same skill after all due skills are done`, async () => {
+    debugger;
+    const reviewQueue = await simulateSkillReviews({
+      targetSkills: [`he:分:divide`],
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)